.github/workflows/python-package.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         python -m pip install types-simplejson types-requests types-click
         python -m pip install oda-knowledge-base[rdf,cwl]
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-        curl -o tests/oda-ontology.ttl https://raw.githubusercontent.com/oda-hub/ontology/main/ontology.ttl
+        curl -o tests/oda-ontology.ttl https://raw.githubusercontent.com/oda-hub/ontology/storage_resource_annotations/ontology.ttl
```

Comment:
I think we can merge the ontology now and adapt this.

Reply:
Done